utils.py

```diff
@@ -1047,6 +1061,42 @@ def get_latest_release_url(releases_url):
     return release_url
 
 
+def get_latest_release_version_tag_name(releases_url):
+    data = net_get(releases_url)
+    if data:
+        try:
+            json_data = json.loads(data)
+            logging.debug(f"{json_data=}")
+        except json.JSONDecodeError as e:
+            logging.error(f"Error decoding JSON response: {e}")
+            return None
+
+        if not isinstance(json_data, list) or len(json_data) == 0:
+            logging.error("Invalid or empty JSON response.")
+            return None
+    else:
+        logging.critical("Could not get latest release URL.")
+        return None
+
+    release_tag_name = json_data[0].get('tag_name')  # noqa: E501
+    logging.info(f"Release URL Tag Name: {release_tag_name}")
+    return release_tag_name
+
+
+def set_logoslinuxinstaller_latest_release_config():
+    releases_url = "https://api.github.com/repos/FaithLife-Community/LogosLinuxInstaller/releases"  # noqa: E501
+    logoslinuxinstaller_url = get_latest_release_url(releases_url)
+    logoslinuxinstaller_tag_name = get_latest_release_version_tag_name(releases_url)
```

Comment:
Efficiency suggestion: Both `get_latest_release_*` functions grab the JSON response from the `releases_url`. Can we streamline this by having a single function, e.g. `get_latests_release_data`, that gets the JSON data in one request, then we parse it two different ways to get the `logoslinuxinstaller_url` and the `logoslinuxinstaller_tag_name`? I'd suggest that `get_latest_release_url` and `get_latest_release_version_tag_name` just take the JSON data as the parameter, rather than the URL in that case. (This is a small benefit, but people on slow connections like me really notice these things.)